AEPCore/Sources/eventhub/EventHub.swift

```diff
@@ -261,13 +261,18 @@ final class EventHub {
             return nil
         }
 
-        var version = 0 // default to version 0
+        guard let sharedState = extensionContainer.sharedState else { return nil }
+
+        // default to version 0
+        var version = 0
         // attempt to version at the event
         if let unwrappedEvent = event, let eventNumber = eventNumberMap[unwrappedEvent.id] {
```

Comment:
we're supposed to shadow the var name when unwrapping according to our style guide, right?